sonic-thermalctld/scripts/thermalctld

```diff
@@ -515,9 +524,13 @@ class TemperatureUpdater(logger.Logger):
         low_threshold = NOT_AVAILABLE
```

Comment:
high_threshold not initialized

Reply:
It is initialized in the file view. May be the diffs does not show it.